runtime/doc/lsp.txt

```diff
@@ -1589,12 +1589,16 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
 Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
 
 enable({enable}, {filter})                       *vim.lsp.inlay_hint.enable()*
-    Enables or disables inlay hints for a buffer.
+    Enables or disables inlay hints for the {filter}ed scope.
 
-    To "toggle", pass the inverse of `is_enabled()`: >lua
+    To "toggle" globally, pass the inverse of `is_enabled()`: >lua
```

Comment:
this sentence isn't about scope, it's about toggling. toggling is always done with is_enabled, regardless of scope.

the example happens to set global scope, but that's irrelevant.
```suggestion
    To "toggle", pass the inverse of `is_enabled()`: >lua
```